diff --git a/src/imageformats/xcf.cpp b/src/imageformats/xcf.cpp
index 9a20cbe..7d8bccf 100644
--- a/src/imageformats/xcf.cpp
+++ b/src/imageformats/xcf.cpp
@@ -960,7 +960,11 @@ bool XCFImageFormat::loadImageProperties(QDataStream &xcf_io, XCFImage &xcf_imag
         case PROP_PARASITES:
             while (!property.atEnd()) {
                 char *tag;
+#if QT_VERSION < QT_VERSION_CHECK(6, 7, 0)
                 quint32 size;
+#else
+                qsizetype size;
+#endif
 
                 property.readBytes(tag, size);
 
 
