--- PKGBUILD.orig	2024-01-18 13:19:45.806899664 +0000
+++ PKGBUILD	2024-01-18 13:21:12.352553411 +0000
@@ -46,9 +46,9 @@
 ##        kde.pam
 ##        kde-fingerprint.pam
 ##        kde-smartcard.pam)
-        kde.pam
-        kde-fingerprint.pam 
-        kde-smartcard.pam)
+#        kde.pam
+#        kde-fingerprint.pam 
+#        kde-smartcard.pam)
 #sha256sums=('e568840af812976eae4ccc4dc8c42c80dce6950e120dec26ac409d49b380a597'
 #            'SKIP'
 #            'adba7bb7c27eb3a572e5e9d3cea0dbeebe59d3634472d1863d14fe892cb13b2b'
@@ -75,11 +75,11 @@
 }
 
 # sums-already-applied
-sha256sums=('SKIP')
+sha256sums=('SKIP' 'SKIP' 'SKIP' 'SKIP')
     
 # source-already-applied
-source=(git+https://github.com/KDE/${pkgname}#branch=master)
-    
+source=(git+https://github.com/KDE/${pkgname}#branch=master kde.pam kde-fingerprint.pam kde-smartcard.pam)
+
 # pkgver-already-applied
 
 pkgver() {
