--- PKGBUILD.orig	2023-12-21 16:39:33.053698839 +0000
+++ PKGBUILD	2023-12-21 16:48:43.115508030 +0000
@@ -46,9 +46,6 @@
 ##        kde.pam
 ##        kde-fingerprint.pam
 ##        kde-smartcard.pam)
-        kde.pam
-        kde-fingerprint.pam 
-        kde-smartcard.pam)
 #sha256sums=('b92084283bf477d1fd1c9ce5de2356d02297c9e4f7e955ae267d16fa61e2e26b'
 #            'SKIP'
 #            'adba7bb7c27eb3a572e5e9d3cea0dbeebe59d3634472d1863d14fe892cb13b2b'
@@ -75,11 +72,11 @@
 }
 
 # sums-already-applied
-sha256sums=('SKIP')
+sha256sums=('SKIP' 'SKIP' 'SKIP' 'SKIP')
     
 # source-already-applied
-source=(git+https://github.com/KDE/${pkgname}#branch=master)
-    
+source=(git+https://github.com/KDE/${pkgname}#branch=master kde.pam kde-fingerprint.pam kde-smartcard.pam)
+
 # pkgver-already-applied
 
 pkgver() {
