--- PKGBUILD.orig	2023-12-21 16:57:32.150881761 +0000
+++ PKGBUILD	2023-12-21 16:57:03.767454441 +0000
@@ -55,7 +55,7 @@
 pkgver() {
 	cd ${srcdir}/${pkgname}
 	#git describe --long --tags --abbrev=7 | sed 's/\([^-]*-g\)/r\1/;s/-/./g;s/^v//'
-    echo "$(git describe --long --tags --abbrev=7 | sed 's/\([^-]*-g\)/r\1/;s/-/./g;s/^v//')_$(date +%y%m%d)"
+    echo "6.beta.$(git describe --long --tags --abbrev=7 | sed 's/\([^-]*-g\)/r\1/;s/-/./g;s/^v//')_$(date +%y%m%d)"
 }
 
 
